content/ecosystem.html

```diff
@@ -170,19 +173,105 @@ <h2 id="connectors">Connectors</h2>
 
 <p>Connectors provide code for interfacing with various third-party systems.</p>
 
-<p>Currently these systems are supported:</p>
-
-<ul>
-  <li><a href="https://ci.apache.org/projects/flink/flink-docs-release-1.7/dev/connectors/kafka.html" target="_blank">Apache Kafka</a> (sink/source)</li>
-  <li><a href="https://ci.apache.org/projects/flink/flink-docs-release-1.7/dev/connectors/elasticsearch.html" target="_blank">Elasticsearch 1.x / 2.x / 5.x / 6.x</a> (sink)</li>
-  <li><a href="https://ci.apache.org/projects/flink/flink-docs-release-1.7/dev/connectors/filesystem_sink.html" target="_blank">HDFS</a> (sink)</li>
-  <li><a href="https://ci.apache.org/projects/flink/flink-docs-release-1.7/dev/connectors/rabbitmq.html" target="_blank">RabbitMQ</a> (sink/source)</li>
-  <li><a href="https://ci.apache.org/projects/flink/flink-docs-release-1.7/dev/connectors/kinesis.html" target="_blank">Amazon Kinesis Streams</a> (sink/source)</li>
-  <li><a href="https://ci.apache.org/projects/flink/flink-docs-release-1.7/dev/connectors/twitter.html" target="_blank">Twitter</a> (source)</li>
-  <li><a href="https://ci.apache.org/projects/flink/flink-docs-release-1.7/dev/connectors/nifi.html" target="_blank">Apache NiFi</a> (sink/source)</li>
-  <li><a href="https://ci.apache.org/projects/flink/flink-docs-release-1.7/dev/connectors/cassandra.html" target="_blank">Apache Cassandra</a> (sink)</li>
-  <li><a href="https://github.com/apache/bahir-flink" target="_blank">Redis, Flume, and ActiveMQ (via Apache Bahir)</a> (sink)</li>
-</ul>
+<p>
```

Comment:
I think it would be good to make a pass over the third party projects and remove all that are not in a reasonable shape. For example the Cascading integration should be removed and there are probably others that are not maintained. Linking them does not provide any benefit and might even give the feeling that the page is not maintained.